rust/sbp/src/wire_format.rs

```diff
@@ -0,0 +1,210 @@
+//! Functionality for parsing/writing SBP message payloads with in memory buffers
```

Comment:
Some actual new-ish code. A combination of `SbpSerialize` and all the free standing `parse_<type>()` functions